File: Backend/Models/Reports.cs
namespace Backend.Models
{
  public class SalePerClient
  {
    public int clientID { get; set; }
    public string clientName { get; set; }
    public string clientLastName1 { get; set; }
    public string clientLastName2 { get; set; }
    public int totalPurchases { get; set; }
    public string storeName { get; set; }
    public string delManName { get; set; }
    public string delManLastName1 { get; set; }
    public string delManLastName2 { get; set; }
    public int totalPrice { get; set; }
  }

  public class SalePerStore
  {
    public int storeID { get; set; }
    public string storeName { get; set; }
    public int totalSales { get; set; }
    public int totalPrice { get; set; }
  }
}

File: Frontend/src/app/Interfaces/Reports.ts

interface SalePerClient {
  clientID: number
  clientName: string
  clientLastName1: string
  clientLastName2: string
  totalPurchases: number
  storeName: string
  delManName: string
  delManLastName1: string
  delManLastName2: string
  totalPrice: number
}

interface SalePerStore {
  storeID: number
  storeName: string
  totalSales: number
  totalPrice: number
}

export {
  SalePerClient,
  SalePerStore
}